### ISSUE TITLE
Station ID..

### ISSUE THREAD

**Issue by jpronans** [claudobahn/weewx-windguru#2]
Might be worth mentioning that the Station ID being referred to in the README is the "UID" shown in the "Station Status" page. It took me a short while to figure that out.

**Comment by claudobahn** [comment]
@jpronans talking about the WeeWX ID here?
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/2065015/179995841-d39d4777-539d-412f-aa7e-06a8eb8d521e.png">

anything else that you found lacking in the README?

**Comment by jpronans** [comment]
Yes, thats it, so after you go past that page it is no longer called WeeWX ID.  Which was a tad confusing. Nothing you can do about it except mention it in the README.  Thanks for the plugin.
